i18n-embed/src/fluent.rs

```diff
@@ -0,0 +1,73 @@
+use crate::{domain_from_module, LanguageLoader};
+use fluent::FluentValue;
+use std::{collections::HashMap, sync::RwLock};
+use unic_langid::LanguageIdentifier;
+
+lazy_static::lazy_static! {
+    static ref CURRENT_LANGUAGE: RwLock<LanguageIdentifier> = {
+        let language = LanguageIdentifier::default();
+        RwLock::new(language)
```

Comment:
Refactor to use the improved [parking_lot::RwLock](https://docs.rs/parking_lot/0.11.0/parking_lot/type.RwLock.html)